### DISCUSSION TITLE
【丰富 Legend 交互】原：Legend支持selectedMode吗？

### DISCUSSION THREAD

**Discussion by jlongguo** [antvis/G2Plot#2330] [Q&A] [answered]
G2是支持图例两种选择模式的'single' | 'multiple'，
什么时候支持呀

**Comment by visiky** [comment, marked as answer]
1⃣️  legend 支持单选交互，见: https://github.com/antvis/G2/pull/3289 使用方式

2⃣️ 问题

Q：多折线图，如何在初始化时指定隐藏折线？触发点击legend的隐藏？
A：手动在 plot 渲染之后，调用 g2 chart 的 filter 方法，如下：
```ts
plot.chart.filter('area', area => area !== '华北')
```
